src/main/java/org/apache/cassandra/sidecar/routes/HealthCheck.java

```diff
@@ -64,7 +64,7 @@ private boolean check()
                 return result;
             }
         }
-        catch (NoHostAvailableException nha)
+        catch (NoHostAvailableException | ConnectionException nha)
         {
             logger.trace("NoHostAvailableException in HealthCheck - Cassandra Down");
```

Comment:
This message needs to change. Could you also add the exception to the logger so it prints the stacktrace in the logs?